### ISSUE TITLE
Form elements should have labels in slide-panel-manager examples

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#509]
# Issue

## Description
Form elements should have labels in slide-panel-manager examples

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-ui/#/components/terra-slide-panel-manager/slide-panel-manager/slide-panel-manager
2. Run the aXe chrome extensions to view the accessibility warnings

## Expected Behavior
Slide panel manager docs follow accessibility guidelines.

## Possible Solution
Use labels for with inputs

## Environment

* Component Name and Version: terra-slide-panel-manager

**Comment by AlexBrizi** [comment]
Should be resolved as a result of [#544](https://github.com/cerner/terra-framework/pull/544)
